public/app/core/components/NavBar/NavBar.tsx

```diff
@@ -42,14 +41,17 @@ export const NavBar: FC = React.memo(() => {
   const theme = useTheme2();
   const styles = getStyles(theme);
   const location = useLocation();
+  const { sttEnabled, alertingEnabled, dbaasEnabled, backupEnabled } = useSelector(
```

Comment:
Should we have a file to put all of our selectors? Grafana has a bunch of those if you search for selector.ts

Reply:
Yeah, agree. But do you think these are different selectors, or just one? Because all flags are part of the `perconaSettings` slice.

Reply:
I would create two, one for the settings and one for the user. And the rest like the flags we can access directly from the return of the settings selector

Reply:
Makes sense !